### ISSUE TITLE
fine-tuning XLM for NER, loss decreases but F1 score remain unchanged

### ISSUE THREAD

**Issue by stefensa** [facebookresearch/XLM#132]
Hi, I want to fine-tuning XNLI-15 pertained model for NER downstream task, so i added a BiLSTM and CRF architecture.

But a weird thing happened, the loss of my model keep decreasing, but the F1 score remain unchanged in an initial status at epoch0, iteration 1. I don't know the reason about it,  anyone met this phenomenon before? Appreciate it for your help.
This is the initial state of my model performance.
<img width="714" alt="WechatIMG37" src="https://user-images.githubusercontent.com/26850639/61370397-e628ca00-a8c4-11e9-86d4-d1a0876bd66a.png">

And this is the performance after 9 epoch and 1840 iterations(batch_size is 32)
<img width="557" alt="WechatIMG36" src="https://user-images.githubusercontent.com/26850639/61370453-048ec580-a8c5-11e9-87ce-9f644be413ac.png">

My code have been uploaded to GitHub. 
[https://github.com/stefensa/XLM_NER](url)
Just ```python  (your_file_dir)/fine_tuning/ner.py ```to begin training.
Moreover, the pertained model I used is from XLM 15 language MLM + TLM model [https://dl.fbaipublicfiles.com/XLM/mlm_tlm_xnli15_1024.pth](url)

Ps: The code for CRF is from [https://pytorch-crf.readthedocs.io/en/stable/#api-documentation](url)   So it need to be installed by ```pip install pytorch-crf```

**Comment by stefensa** [comment]
I just spend a whole day to figure out the reason of this bug but I failed, thanks for any help.
